provisionerd/provisionerd.go

```diff
@@ -299,7 +300,10 @@ func (p *Server) acquireLoop() {
 			p.opts.Logger.Debug(ctx, "shut down before client (re) connected")
 			return
 		}
-		p.acquireAndRunOne(client)
+		err := p.acquireAndRunOne(client)
+		if err != nil && ctx.Err() == nil { // Only log if context is not done.
+			p.opts.Logger.Debug(ctx, "retrying to acquire job", slog.F("retry_in_ms", retrier.Delay.Milliseconds()), slog.Error(err))
```

Comment:
Self-review: `acquireAndRunOne` already logs its own warning - specifically the `provisionerd was unable to acquire job` one is logged when the db is unreachable - so `Debug` is what felt most appropriate to me.